src/Saas.Permissions/Saas.Permissions.Api/Controllers/PermissionsController.cs

```diff
@@ -21,6 +24,7 @@ public PermissionsController(IPermissionsService permissionsService)
     [ProducesResponseType(StatusCodes.Status400BadRequest)]
     [ProducesResponseType(StatusCodes.Status401Unauthorized)]
     [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    [RequiredScope(new[] { "read_via_admin_api", "write_via_admin_api" })]
```

Comment:
@asanjabi This better? I chose to accept either read or write for any read queries. Thought it might be easier than requesting both scopes, but let me know if you have a preference on that.

Reply:
@landonpierce Much better, however need to remove "via_admin_api" we don't really want scopes based on another service.  Leave it at "permissions.read" "permissions.write"  who knows what application we might give access to.  Take a look at our graph API permissions, it's a good example.
https://docs.microsoft.com/en-us/graph/permissions-reference#application-permissions